lambdas/service/app.py

```diff
@@ -533,6 +537,96 @@ def get_manifest():
     return response
 
 
+@app.route('/manifest/files', methods=['GET'], cors=True)
+def start_manifest_generation():
+    """
+    Start and check status of manifest generation jobs
+
+    parameters:
+        - name: filters
+          in: query
+          type: string
+          description: Filters to be applied when generating the manifest
+        - name: token
+          in: query
+          type: string
+          description: An opaque string describing the manifest generation job
+    :return: Response with either a 302 status and the location of the generated manifest,
+        or a 301 status and a URL to re-check the status of the job
+
+        If manifest is still generating the response will look like:
+        {
+            "Status": 301,
+            "Retry-After": 2,
+            "Location": "https://retry.url"
+        }
+        Status represents an HTTP status code (301 moved permanently)
+        Retry-After is the recommended number of seconds to wait before rechecking the status of the manifest
+        Location is the URL to make a GET request to in order to recheck the status
```

Comment:
Empty lines between paragraphs and periods at the end of sentences. Code literals like `Location` should be in back ticks. Preformatted sections like JSON fragments should be between triple back-ticks.